bumpCversion.py

```diff
@@ -133,17 +169,36 @@ def main():
     # Parse command line arguments
     args = parse_args()
 
-    # users desired part (major, minor, or patch) to bump
+    # Users desired part (major, minor, or patch) to bump
     part_to_bump = args.part
 
-    # get file we're interested in
+    # Get file we're interested in
     target_files = get_target_files(args)
 
+    # Create list of found versions (length should be == 1)
+    versions = set()
+
+    # Check for dry run
+    if args.dry_run:
+        print_dry(args, target_files.copy(), versions)
+        exit()
+
+    # Check to see if version number is the same across all files
+    print("Checking component [" + str(args.component) +
+          "] for verison number congruence...\n")
+    valid_version_congruence(args, target_files.copy(), versions)
```

Comment:
Would it make sense to call `valid_version_congruence` *before* we check for the dry run argument? 

The reason I ask, is because `valid_version_congruence` is called in `print_dry`. Let me know what you think.

Reply:
We can leave the function call inside of `print_dry` or call it before `print_dry`. Not too sure which is better. I think main is cleaner as it stands, but calling `valid_version_congruence` in main would explicitly let us know what were doing without having to look at the details of `print_dry`.